src/pages/resources/_exercises/css4js.md

```diff
@@ -16,6 +16,7 @@
   - [Sole and Ankle Revisited - responsive design](https://github.com/css-for-js/sole-and-ankle-revisited)
   - [Unsprinkle - typography and images](https://github.com/css-for-js/unsprinkle)
 
+
```

Comment:
Nothing changed

Reply:
Just ignore it - it shouldn't be in this PR.